From f6f96636180480800038ea05deb49512aadb38eb Mon Sep 17 00:00:00 2001
From: hjy0951 <hjy0951@gmail.com>
Date: Thu, 19 Jun 2025 14:53:14 +0900
Subject: [PATCH 1/5] =?UTF-8?q?docs:=20README=20=EC=97=85=EB=8D=B0?=
 =?UTF-8?q?=EC=9D=B4=ED=8A=B8?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 README.md | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/README.md b/README.md
index b992406..302ce3c 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-### Heojooon.
+### HEOJOOON.
 
 > 개발 및 일상을 다루는 블로그
 
@@ -22,7 +22,6 @@
   - [x] 프로필 카드 만들기
   - [x] 일상 / 개발 글 섹션 분리
   - [x] 키워드별 모아보기
+- [x] 다크모드
 - [ ] SEO 최적화
 - [ ] 이력 페이지 (about)
-- [ ] 사진 일기 페이지
-- [ ] 다크모드
-- 
2.41.0

